### PR TITLE
LPD-42616 - Update SF for copyright header

### PR DESCRIPTION
**Pull request by bryceosterhaus**

### CODE REVIEW

File: modules/.eslintrc.js [hunk 1]
```diff
@@ -85,7 +85,7 @@ const config = {
 		'notice/notice': [
 			'error',
 			{
-				nonMatchingTolerance: 0.7,
+				nonMatchingTolerance: 0.95,
```
Comment (izaera) [inline modules/.eslintrc.js:88]: Why not 100%? :thinking:

**Review by izaera — APPROVED** [review]
LGTM, only the doubt about tolerance... :thinking: 

You mentioned in slack using 100% ?

File: modules/.eslintrc.js (hunk 1 above)
Comment (bryceosterhaus) [reply modules/.eslintrc.js:88]: There are weird issues with trailing new lines and it would add repeated new lines at the end of the header.

**Comment by bryceosterhaus** [comment]
ci:report:74063432

**Comment by bryceosterhaus** [comment]
ci:forward

**Comment by bryceosterhaus** [comment]
ci:stop

**Comment by bryceosterhaus** [comment]
ci:forward:force

**Pull request closed without merging**